docs/source/tutorials/datalab/workflows.ipynb

```diff
@@ -549,6 +556,245 @@
     "Learn more about the underperforming group issue type [here](issue_type_description.html#underperforming-group-issue)."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using Predefined Data Slices to Detect Underperforming Groups\n",
+    "\n",
+    "In addition to the clustering approach, you can use predefined data slices to detect underperforming groups, similar to what we did in the section on [detecting underperforming groups](#Find-Underperforming-Groups-in-a-Dataset).\n",
+    "\n",
+    "By default, Datalab will automatically detect underperforming groups when you provide numerical embeddings and predicted probabilities, clustering them based on the provided embeddings.\n",
+    "\n",
+    "However, for a tabular dataset, you can use the values of a categorical column as cluster IDs to define specific groups. This approach allows you to focus on meaningful slices of your data that are defined by domain knowledge or specific attributes."
```

Comment:
```suggestion
    "## Predefining Data Slices for Detecting Underperforming Groups\n",
    "\n",
    "Instead of clustering the data to determine what data slices are considered when detecting underperforming groups, you can define these slices yourself.\n",
    "For say a tabular dataset, you can use the values of a categorical column as cluster IDs to predefine the relevant data subgroups/slices to consider. This allows you to focus on meaningful slices of your data defined by domain knowledge or specific attributes."
```